### DISCUSSION TITLE
textadept curses in linux console

### DISCUSSION THREAD

**Discussion by eda2z** [orbitalquark/textadept#137] [Q&A] [answered]
I realize that most people are running gui's of some sort, but when I build a new linux server I do not install X. I noticed that curses based textadept runs fine in a terminal emulator running under X but a lot of functionality is missing when I am running on the linux console (i.e. TERM=linux). Is this intended or am I missing something. Thanks.

**Comment by orbitalquark** [comment]
Is there anything missing from the list of known constraints? https://orbitalquark.github.io/textadept/manual.html#terminal-version-compatibility

**Comment by orbitalquark** [comment, marked as answer]
Sorry, I read your question wrong. The default Linux tty lacks a lot of features, so I'm not surprised you're seeing a lack of features compared to a terminal emulator in a GUI.
